### PR TITLE
update README to include a quick guide

### PR DESCRIPTION
**Pull request by zachovich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,16 @@ This is the main repo for the GreenLake terraform provider which provides terraf
 We're currently in the process of open sourcing all the code, meanwhile, this repository holds the following:
 
 1. HPEGL release binaries,
-1. HPEGL documentation,
-1. A script to automate the provider installation for Linux.
+1. HPEGL provider [documentaion](docs/)
```
Comment (howleyt) [inline README.md:10]: typo: documentation

File: README.md [hunk 2]
```diff
@@ -7,10 +7,16 @@ This is the main repo for the GreenLake terraform provider which provides terraf
...
+1. A set of scripts to automate the provider installation.
 
-## Installation
+## Prerequisite
```
Comment (howleyt) [inline README.md:13]: Prerequisites

File: README.md [hunk 3]
```diff
@@ -7,10 +7,16 @@ This is the main repo for the GreenLake terraform provider which provides terraf
...
+
+1. Terraform version >= v0.13 [install terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+1. A Service Client to authenticate against GreenLake.
```
Comment (howleyt) [inline README.md:16]: I guess we'll be inserting a link to docs on service client retrieval when it's available?

**Review by howleyt — COMMENTED** [review]
Looks good, couple of minor things.

File: README.md (hunk 3 above)
Comment (zachovich) [reply README.md:16]: @howleyt that's the plan

**Review by J0HNB0Y — APPROVED** [review]
LGTM

**Pull request merged**